src/components/WebexMemberRoster/WebexMemberRoster.scss

```diff
@@ -16,4 +16,8 @@
       margin-left: 0.625rem;
     }
   }
+
+  .participants-counter{
```

Comment:
space! :)

Reply:
put this higher up so that the order of the classes in this file matches as much as possible the order of the elements in dom